src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinInGameHud.java

```diff
@@ -111,6 +115,28 @@ private void hookFreeCamRenderCrosshairInThirdPerson(DrawContext context, Render
         }
     }
 
+    @Redirect(method = "renderCrosshair", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/DrawContext;drawGuiTexture(Ljava/util/function/Function;Lnet/minecraft/util/Identifier;IIII)V", ordinal = 0))
```

Comment:
Please do not use `@Redirect`. Instead of e.g. `@WrapOperation`, `@ModifyExpressionValue` ... etc